packages/cloud-config/build.yaml

```diff
@@ -7,19 +7,23 @@ requires:
   version: ">=0"
 
 steps:
-- mkdir -p /system /oem
-- cp -rfv oem /system
-- chmod -R 600 /system/oem
-- cos-tmpl-render /system/oem/00_rootfs.yaml.tmpl
-- cos-tmpl-render /system/oem/06_recovery.yaml.tmpl
+- mkdir -p {{.Values.oem_dir}}
+- cp -rfv oem/{{.Values.oem_file}} {{.Values.oem_dir}}
```

Comment:
Why don't we copy files based on the package name? This way we don't need the `oem_file` template, we just pick the require file (or multiple files) in each package.

IMHO this way is more scalable and easier to read.

Reply:
well, they have numbers in front and I avoided to do any smart code here. I thought it would be quite unreadable to have numbers in front of the package name, and wildcard.. well can have unexpected consequences.